_summaries/gropp2020implicit_2.md

```diff
@@ -1,19 +1,85 @@
 ---
 layout: summary
 title: Summary
-paper: {{paper_tag}}
-# Please fill out info below
-author: # Your GitHub id
-score: # How did you like this paper 0(dislike) to 10(love)
+paper: gropp2020implicit
+author: # samatharhay
+score: # 7
 ---
 
-TODO: Summarize the paper:
 * What is the core idea?
+
+The core idea of the paper was to introduce a new paradigm built on implicit geometric regularization for computing high fidelity implicit neural representations directly from raw data scans.
+
 * How is it realized (technically)?
+
+Given an input cloud X = {x_i}iEI C R^3 with or without normal data, N = {n_i} iEI C R^3, the goal is to compute the parameters \theta of a multi-layer perceptron (MLP) f(x;\theta), where f has the shape R^3 x R^m \rightarrow R, such that it approximates a signed distance function to a plausible surface M defined by X and N.
+
+To help learn these parameters the authors propose the loss function:
+
+![image of function 2](gropp2020implicit_2a.png)
+
+where \lambda > 0 is a hyperparameter, ||.|| = ||.||2 (the euclidean 2-norm) and 
```

Comment:
Use $$\\| . \\|$$ for the bars, otherwise it'll look like this:

![image](https://user-images.githubusercontent.com/22307142/137611057-24f15384-2b5b-4277-aade-515b8436c504.png)